libse/Forms/FixCommonErrors/FixMissingSpaces.cs

```diff
@@ -237,12 +237,11 @@ public void Fix(Subtitle subtitle, IFixCallbacks callbacks)
                 if (index >= 0 && p.Text.Length > 5)
                 {
                     string newText = p.Text;
-                    while (index != -1)
+                    while (index >= 0)
                     {
                         if (newText.Length > index + 6 && index > 1)
                         {
-                            if (Utilities.AllLettersAndNumbers.Contains(newText[index + 3]) &&
-                                Utilities.AllLettersAndNumbers.Contains(newText[index - 1]))

```

Comment:
Character before italic-start tag could be another tag of one of the following: `.`, `?`, `!`.